src/complex.jl

```diff
@@ -0,0 +1,172 @@
+Base.prevfloat(x::Dual{T,V}) where {T,V<:AbstractFloat} = prevfloat(x.value)
```

Comment:
see https://github.com/JuliaDiff/ForwardDiff.jl/pull/453

Reply:
Thanks. Is that PR going to be merged soon? If so, feel free to merge that PR first, I can do the compatibility check later.

Reply:
done